report_builder/utils.py

```diff
@@ -31,3 +34,65 @@ def duplicate(obj, changes=None):
             except:
                 pass
     return duplicate
+
+
+DATE = 1
```

Comment:
would an enum work here instead of magic numbers? https://docs.python.org/3/library/enum.html